[INFO] Error stacktraces are turned on.
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for com.github.sepulrator:exception-util:jar:1.0.0
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-source-plugin is missing. @ line 97, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] Inspecting build with total of 1 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 1 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
[INFO] 
[INFO] ----------------< com.github.sepulrator:exception-util >----------------
[INFO] Building exception-util 1.0.0
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:prepare-agent (default) @ exception-util ---
[INFO] argLine set to -javaagent:/home/jdbl/.m2/repository/org/jacoco/org.jacoco.agent/0.8.5/org.jacoco.agent-0.8.5-runtime.jar=destfile=/tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ exception-util ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ exception-util ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 13 source files to /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/target/classes
[WARNING] /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/src/main/java/org/javaex/exception/handler/SingletonClientExceptionHandler.java: /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/src/main/java/org/javaex/exception/handler/SingletonClientExceptionHandler.java uses unchecked or unsafe operations.
[WARNING] /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/src/main/java/org/javaex/exception/handler/SingletonClientExceptionHandler.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- aspectj-maven-plugin:1.7:compile (default) @ exception-util ---
[INFO] Showing AJC message detail for messages of types: [error, warning, fail]
[INFO] Join point 'method-call(org.slf4j.Logger org.slf4j.LoggerFactory.getLogger(java.lang.Class))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:22) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean org.slf4j.Logger.isDebugEnabled())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:36) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.debug(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:37) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.aspectj.lang.Signature org.aspectj.lang.JoinPoint.getSignature())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:40) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.aspectj.lang.Signature.getDeclaringTypeName())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:41) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.aspectj.lang.Signature.getName())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:42) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object[] org.aspectj.lang.JoinPoint.getArgs())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:43) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.util.Arrays.toString(java.lang.Object[]))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:43) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StackTraceElement[] java.lang.Throwable.getStackTrace())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:45) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StackTraceElement.getMethodName())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:47) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.String.equals(java.lang.Object))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:47) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StackTraceElement.getClassName())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.String.equals(java.lang.Object))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean org.slf4j.Logger.isDebugEnabled())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:52) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:54) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:54) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:55) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:55) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:55) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.Throwable.toString())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:56) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:56) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:56) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.debug(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:53) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.ExceptionInfo org.javaex.exception.ExceptionInfo.setException(java.lang.Throwable))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:61) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Class java.lang.Object.getClass())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:62) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.Class.toString())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:62) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.ExceptionInfo org.javaex.exception.ExceptionInfo.setName(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:62) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.ExceptionInfo org.javaex.exception.ExceptionInfo.setClassName(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:63) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StackTraceElement.getFileName())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:64) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.ExceptionInfo org.javaex.exception.ExceptionInfo.setFileName(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:64) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(int java.lang.StackTraceElement.getLineNumber())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:65) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.ExceptionInfo org.javaex.exception.ExceptionInfo.setLineNumber(int))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:65) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.ExceptionInfo org.javaex.exception.ExceptionInfo.setMethodName(java.lang.String))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:66) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object[] org.aspectj.lang.JoinPoint.getArgs())' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:67) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.List java.util.Arrays.asList(java.lang.Object[]))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:67) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.ExceptionInfo org.javaex.exception.ExceptionInfo.setArguments(java.util.List))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:67) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.exception.publisher.ExceptionEventPublisher.publishEvent(org.javaex.exception.ExceptionInfo))' in Type 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:70) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.slf4j.Logger org.slf4j.LoggerFactory.getLogger(java.lang.Class))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:23) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.client.ClientExceptionAdviceClassInfo.setClassInfo())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:26) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.client.ClientExceptionAdviceClassInfo.setMethodInfo())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:27) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.javaex.client.ClientExceptionAdviceClassInfo.getExceptionAdviceAnnotatedClassName())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.String.equals(java.lang.Object))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:32) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.warn(java.lang.String))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:33) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.info(java.lang.String))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:36) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.reflect.Method[] java.lang.Class.getDeclaredMethods())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:42) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.List java.util.Arrays.asList(java.lang.Object[]))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:42) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.Iterator java.util.List.iterator())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:44) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object java.util.Iterator.next())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:44) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.reflect.Method.isAnnotationPresent(java.lang.Class))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:45) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.client.ClassMethodInfo.setMethod(java.lang.reflect.Method))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:47) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.annotation.Annotation java.lang.reflect.Method.getAnnotation(java.lang.Class))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:49) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Class[] org.javaex.annotation.ExceptionHandler.value())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:51) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.List java.util.Arrays.asList(java.lang.Object[]))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:51) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.client.ClassMethodInfo.setExceptionList(java.util.List))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:52) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.reflect.Method.isAnnotationPresent(java.lang.Class))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:54) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.annotation.Annotation java.lang.reflect.Method.getAnnotation(java.lang.Class))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:55) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String[] org.javaex.annotation.ExceptionScan.value())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:56) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.List java.util.Arrays.asList(java.lang.Object[]))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:56) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.client.ClassMethodInfo.setPackageList(java.util.List))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:57) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.client.ClassMethodInfo.setPackageList(java.util.List))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:59) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.util.List.add(java.lang.Object))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:62) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.util.Iterator.hasNext())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:44) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.Set org.reflections.Reflections.getTypesAnnotatedWith(java.lang.Class, boolean))' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:70) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(int java.util.Set.size())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:71) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object[] java.util.Set.toArray())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:74) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.Class.getName())' in Type 'org.javaex.client.ClientExceptionAdviceClassInfo' (ClientExceptionAdviceClassInfo.java:76) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.slf4j.Logger org.slf4j.LoggerFactory.getLogger(java.lang.Class))' in Type 'org.javaex.exception.service.FileExceptionDefinitionHandlerService' (FileExceptionDefinitionHandlerService.java:11) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.info(java.lang.String))' in Type 'org.javaex.exception.service.FileExceptionDefinitionHandlerService' (FileExceptionDefinitionHandlerService.java:16) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.javaex.properties.util.ExceptionDefinitionUtil.getDefinition(java.lang.String))' in Type 'org.javaex.exception.service.FileExceptionDefinitionHandlerService' (FileExceptionDefinitionHandlerService.java:22) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.javaex.exception.service.FileExceptionDefinitionHandlerService org.javaex.exception.service.FileExceptionDefinitionHandlerService.getInstance())' in Type 'org.javaex.exception.CustomException' (CustomException.java:15) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.String.trim())' in Type 'org.javaex.exception.CustomException' (CustomException.java:20) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.CustomException' (CustomException.java:20) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.CustomException' (CustomException.java:24) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.CustomException' (CustomException.java:27) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.javaex.exception.service.DefinitionHandlerService.getErrorMessage(java.lang.String))' in Type 'org.javaex.exception.CustomException' (CustomException.java:40) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.slf4j.Logger org.slf4j.LoggerFactory.getLogger(java.lang.Class))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:17) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object org.javaex.exception.handler.SingletonClientExceptionHandler.createExceptionListenerInstance(java.lang.String))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:25) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.Exception.getLocalizedMessage())' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:27) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:27) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:27) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.error(java.lang.String))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:27) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.String.valueOf(java.lang.Object))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.info(java.lang.String))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.Set org.reflections.Reflections.getTypesAnnotatedWith(java.lang.Class, boolean))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:42) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(int java.util.Set.size())' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:43) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object[] java.util.Set.toArray())' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:46) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.Class.getName())' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:47) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Class java.lang.Class.forName(java.lang.String))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:53) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.reflect.Constructor java.lang.Class.getConstructor(java.lang.Class[]))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:54) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object java.lang.reflect.Constructor.newInstance(java.lang.Object[]))' in Type 'org.javaex.exception.handler.SingletonClientExceptionHandler' (SingletonClientExceptionHandler.java:55) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.slf4j.Logger org.slf4j.LoggerFactory.getLogger(java.lang.Class))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:14) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object org.javaex.exception.handler.SingletonClientExceptionHandler.getInstance())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:22) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.exception.publisher.ExceptionEventPublisher.invokeExceptionCallBackMethod(org.javaex.exception.ExceptionInfo, java.lang.Object))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:23) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.Exception.toString())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:25) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:25) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:25) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.error(java.lang.String))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:25) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Throwable org.javaex.exception.ExceptionInfo.getException())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:32) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Class java.lang.Object.getClass())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:32) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.Iterator java.util.List.iterator())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:33) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object java.util.Iterator.next())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:33) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.List org.javaex.client.ClassMethodInfo.getExceptionList())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:35) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.Iterator java.util.List.iterator())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:35) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object java.util.Iterator.next())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:35) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.Class.isAssignableFrom(java.lang.Class))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:37) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.List org.javaex.client.ClassMethodInfo.getPackageList())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:39) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.util.List.isEmpty())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:39) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.reflect.Method org.javaex.client.ClassMethodInfo.getMethod())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:40) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:40) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.reflect.Method org.javaex.client.ClassMethodInfo.getMethod())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:41) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.reflect.Method.getName())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:41) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.String.valueOf(java.lang.Object))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:41) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:41) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:41) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.info(java.lang.String))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:41) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.List org.javaex.client.ClassMethodInfo.getPackageList())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:45) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.util.Iterator java.util.List.iterator())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:45) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object java.util.Iterator.next())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:45) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.javaex.exception.ExceptionInfo.getClassName())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:46) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.String.contains(java.lang.CharSequence))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:46) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.reflect.Method org.javaex.client.ClassMethodInfo.getMethod())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:47) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:47) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.reflect.Method org.javaex.client.ClassMethodInfo.getMethod())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.reflect.Method.getName())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.String.valueOf(java.lang.Object))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.info(java.lang.String))' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:48) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.util.Iterator.hasNext())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:45) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.util.Iterator.hasNext())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:35) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(boolean java.util.Iterator.hasNext())' in Type 'org.javaex.exception.publisher.ExceptionEventPublisher' (ExceptionEventPublisher.java:33) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.Object))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:79) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:79) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:79) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:79) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:79) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:80) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:80) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:80) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:80) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:80) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(int))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:80) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:81) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.Object))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:81) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:81) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.exception.ExceptionInfo' (ExceptionInfo.java:79) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(org.slf4j.Logger org.slf4j.LoggerFactory.getLogger(java.lang.Class))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:16) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.properties.util.ExceptionDefinitionUtil.loadDefinitionIntoMemory())' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:20) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.util.Properties.getProperty(java.lang.String))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:22) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.ClassLoader java.lang.Class.getClassLoader())' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:30) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.io.InputStream java.lang.ClassLoader.getResourceAsStream(java.lang.String))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:30) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void java.util.Properties.load(java.io.InputStream))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:33) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.info(java.lang.String))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:34) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.io.IOException.toString())' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:37) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.StringBuilder java.lang.StringBuilder.append(java.lang.String))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:37) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(java.lang.String java.lang.StringBuilder.toString())' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:37) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.slf4j.Logger.warn(java.lang.String))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:37) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.properties.util.ExceptionDefinitionUtil.closeInputStream(java.io.InputStream))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:40) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.properties.util.ExceptionDefinitionUtil.closeInputStream(java.io.InputStream))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:40) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.properties.util.ExceptionDefinitionUtil.closeInputStream(java.io.InputStream))' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:40) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void java.io.InputStream.close())' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:47) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] Join point 'method-call(void java.io.IOException.printStackTrace())' in Type 'org.javaex.properties.util.ExceptionDefinitionUtil' (ExceptionDefinitionUtil.java:49) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.java:34) [with runtime test]
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ exception-util ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 3 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ exception-util ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 6 source files to /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/target/test-classes
[INFO] 
[INFO] --- aspectj-maven-plugin:1.7:test-compile (default) @ exception-util ---
[INFO] Showing AJC message detail for messages of types: [error, warning, fail]
[INFO] Join point 'method-call(void org.javaex.test.util.ExceptionThrower.throwMockException(org.javaex.test.constants.TestConstants$MockException))' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:19) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void java.lang.Exception.printStackTrace())' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:21) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(int[] org.javaex.test.util.ExceptionThrower.$SWITCH_TABLE$org$javaex$test$constants$TestConstants$MockException())' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:26) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(int org.javaex.test.constants.TestConstants$MockException.ordinal())' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:26) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(org.javaex.test.constants.TestConstants$MockException[] org.javaex.test.constants.TestConstants$MockException.values())' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:7) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(int org.javaex.test.constants.TestConstants$MockException.ordinal())' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:7) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(int org.javaex.test.constants.TestConstants$MockException.ordinal())' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:7) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(int org.javaex.test.constants.TestConstants$MockException.ordinal())' in Type 'org.javaex.test.util.ExceptionThrower' (ExceptionThrower.java:7) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.powermock.api.mockito.PowerMockito.mockStatic(java.lang.Class, java.lang.Class[]))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:27) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(org.javaex.test.util.ExceptionThrower org.javaex.test.util.ExceptionThrower.getInstance())' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:34) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.test.util.ExceptionThrower.throwAndHandleException(org.javaex.test.constants.TestConstants$MockException))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:34) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.powermock.api.mockito.PowerMockito.verifyStatic())' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:37) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object org.mockito.Matchers.any(java.lang.Class))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:38) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.test.handler.ClientExceptionHandler.afterNullPointerExceptionThrown(org.javaex.exception.ExceptionInfo))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:38) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(org.javaex.test.util.ExceptionThrower org.javaex.test.util.ExceptionThrower.getInstance())' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:46) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.test.util.ExceptionThrower.throwAndHandleException(org.javaex.test.constants.TestConstants$MockException))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:46) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.powermock.api.mockito.PowerMockito.verifyStatic())' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:49) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object org.mockito.Matchers.any(java.lang.Class))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:50) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.test.handler.ClientExceptionHandler.afterArithmeticExceptionThrownForCorrectPackage(org.javaex.exception.ExceptionInfo))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:50) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(org.mockito.verification.VerificationMode org.mockito.Mockito.never())' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:52) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.powermock.api.mockito.PowerMockito.verifyStatic(org.mockito.verification.VerificationMode))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:52) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(java.lang.Object org.mockito.Matchers.any(java.lang.Class))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:53) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.test.handler.ClientExceptionHandler.afterArithmeticExceptionThrownForWrongPackage(org.javaex.exception.ExceptionInfo))' in Type 'org.javaex.test.integration.ExceptionAdviceCallBackTest' (ExceptionAdviceCallBackTest.java:53) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.test.integration.ExceptionDefinitionMessageTest.throwCustomException())' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:18) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.javaex.test.customexception.MyNullPointerException.getDefinedErrorMessage())' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:21) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.String.equals(java.lang.Object))' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:21) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.junit.Assert.assertTrue(java.lang.String, boolean))' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:21) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.javaex.test.integration.ExceptionDefinitionMessageTest.throwCustomExceptionWithReason())' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:28) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(java.lang.String org.javaex.test.customexception.MyNullPointerException.getDefinedErrorMessage())' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(boolean java.lang.String.equals(java.lang.Object))' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void org.junit.Assert.assertTrue(java.lang.String, boolean))' in Type 'org.javaex.test.integration.ExceptionDefinitionMessageTest' (ExceptionDefinitionMessageTest.java:31) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void java.io.PrintStream.println(java.lang.String))' in Type 'org.javaex.test.handler.ClientExceptionHandler' (ClientExceptionHandler.java:15) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void java.io.PrintStream.println(java.lang.String))' in Type 'org.javaex.test.handler.ClientExceptionHandler' (ClientExceptionHandler.java:22) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void java.io.PrintStream.println(java.lang.String))' in Type 'org.javaex.test.handler.ClientExceptionHandler' (ClientExceptionHandler.java:28) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(void java.lang.System.arraycopy(java.lang.Object, int, java.lang.Object, int, int))' in Type 'org.javaex.test.constants.TestConstants$MockException' (TestConstants.java:1) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] Join point 'method-call(java.lang.Enum java.lang.Enum.valueOf(java.lang.Class, java.lang.String))' in Type 'org.javaex.test.constants.TestConstants$MockException' (TestConstants.java:1) advised by afterThrowing advice from 'org.javaex.exception.handler.ThrowedExceptionAspectHandler' (ThrowedExceptionAspectHandler.class(from ThrowedExceptionAspectHandler.java)) [with runtime test]
[INFO] 
[INFO] --- maven-surefire-plugin:2.19.1:test (default-test) @ exception-util ---
[INFO] Surefire report directory: /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/target/surefire-reports

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.javaex.test.integration.ExceptionDefinitionMessageTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [file:/tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/target/classes/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/jdbl/.m2/repository/org/slf4j/slf4j-log4j12/1.7.22/slf4j-log4j12-1.7.22.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 [main] INFO org.javaex.exception.service.FileExceptionDefinitionHandlerService - File Exception Definition Handler Instantiated
545 [main] INFO org.reflections.Reflections - Reflections took 444 ms to scan 2 urls, producing 24 keys and 731 values 
569 [main] INFO org.javaex.client.ClientExceptionAdviceClassInfo - exception advice found and configured
577 [main] INFO org.javaex.exception.handler.SingletonClientExceptionHandler - org.javaex.test.handler.ClientExceptionHandler is instantiating as a singleton
579 [main] INFO org.javaex.properties.util.ExceptionDefinitionUtil - exception.definition file is found and loaded
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.721 sec - in org.javaex.test.integration.ExceptionDefinitionMessageTest
Running org.javaex.test.integration.ExceptionAdviceCallBackTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [file:/tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/target/classes/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/jdbl/.m2/repository/org/slf4j/slf4j-log4j12/1.7.22/slf4j-log4j12-1.7.22.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
197 [main] INFO org.reflections.Reflections - Reflections took 171 ms to scan 2 urls, producing 24 keys and 731 values 
207 [main] INFO org.javaex.client.ClientExceptionAdviceClassInfo - exception advice found and configured
212 [main] INFO org.javaex.exception.handler.SingletonClientExceptionHandler - org.javaex.test.handler.ClientExceptionHandler is instantiating as a singleton
231 [main] INFO org.javaex.client.ClientExceptionAdviceClassInfo - afterArithmeticExceptionThrownForCorrectPackage method was invoked
java.lang.ArithmeticException
	at org.javaex.test.util.ExceptionThrower.throwMockException(ExceptionThrower.java:30)
	at org.javaex.test.util.ExceptionThrower.throwAndHandleException(ExceptionThrower.java:19)
	at org.javaex.test.integration.ExceptionAdviceCallBackTest.callbackExceptionScanAnnotationTest(ExceptionAdviceCallBackTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:68)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.runTestMethod(PowerMockJUnit44RunnerDelegateImpl.java:310)
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:88)
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:96)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.executeTest(PowerMockJUnit44RunnerDelegateImpl.java:294)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTestInSuper(PowerMockJUnit47RunnerDelegateImpl.java:127)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTest(PowerMockJUnit47RunnerDelegateImpl.java:82)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.runBeforesThenTestThenAfters(PowerMockJUnit44RunnerDelegateImpl.java:282)
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:86)
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:49)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.invokeTestMethod(PowerMockJUnit44RunnerDelegateImpl.java:207)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.runMethods(PowerMockJUnit44RunnerDelegateImpl.java:146)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$1.run(PowerMockJUnit44RunnerDelegateImpl.java:120)
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:33)
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:45)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.run(PowerMockJUnit44RunnerDelegateImpl.java:122)
	at org.powermock.modules.junit4.common.internal.impl.JUnit4TestSuiteChunkerImpl.run(JUnit4TestSuiteChunkerImpl.java:106)
	at org.powermock.modules.junit4.common.internal.impl.AbstractCommonPowerMockRunner.run(AbstractCommonPowerMockRunner.java:53)
	at org.powermock.modules.junit4.PowerMockRunner.run(PowerMockRunner.java:59)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
246 [main] INFO org.javaex.client.ClientExceptionAdviceClassInfo - afterNullPointerExceptionThrown method was invoked
java.lang.NullPointerException
	at org.javaex.test.util.ExceptionThrower.throwMockException(ExceptionThrower.java:28)
	at org.javaex.test.util.ExceptionThrower.throwAndHandleException(ExceptionThrower.java:19)
	at org.javaex.test.integration.ExceptionAdviceCallBackTest.callbackExceptionHandlerAnnotationTest(ExceptionAdviceCallBackTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:68)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.runTestMethod(PowerMockJUnit44RunnerDelegateImpl.java:310)
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:88)
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:96)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.executeTest(PowerMockJUnit44RunnerDelegateImpl.java:294)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTestInSuper(PowerMockJUnit47RunnerDelegateImpl.java:127)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTest(PowerMockJUnit47RunnerDelegateImpl.java:82)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.runBeforesThenTestThenAfters(PowerMockJUnit44RunnerDelegateImpl.java:282)
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:86)
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:49)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.invokeTestMethod(PowerMockJUnit44RunnerDelegateImpl.java:207)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.runMethods(PowerMockJUnit44RunnerDelegateImpl.java:146)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$1.run(PowerMockJUnit44RunnerDelegateImpl.java:120)
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:33)
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:45)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.run(PowerMockJUnit44RunnerDelegateImpl.java:122)
	at org.powermock.modules.junit4.common.internal.impl.JUnit4TestSuiteChunkerImpl.run(JUnit4TestSuiteChunkerImpl.java:106)
	at org.powermock.modules.junit4.common.internal.impl.AbstractCommonPowerMockRunner.run(AbstractCommonPowerMockRunner.java:53)
	at org.powermock.modules.junit4.PowerMockRunner.run(PowerMockRunner.java:59)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.22 sec - in org.javaex.test.integration.ExceptionAdviceCallBackTest

Results :

Tests run: 4, Failures: 0, Errors: 0, Skipped: 0

[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.5:report (report) @ exception-util ---
[INFO] Loading execution data file /tmp/tmpmeclwiiq/java-exception-api/JAVA_Exception_API/target/jacoco.exec
[INFO] Analyzed bundle 'exception-util' with 2298 classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  22.294 s
[INFO] Finished at: 2020-07-30T22:37:44Z
[INFO] ------------------------------------------------------------------------
